src/airclerk/main.py

```diff
@@ -222,20 +222,40 @@ async def login(request: air.Request, next: str = "/"):
 
 
 @router.post(settings.CLERK_LOGOUT_ROUTE)
-async def logout(request: air.Request, user=require_auth):
+async def logout(request: air.Request):
     """
     Return a page that triggers client-side logout via Clerk JavaScript SDK
     This will clear the JWT token from browser cookies
     """
-    return air.Script(
-        f"""
-            document.addEventListener('htmx:load', async () => {{
-                if (!window.Clerk) return;
-                
-                await window.Clerk.load();
-                
-                // Sign out on the client side (clears cookies/tokens)
-                await window.Clerk.signOut({{ redirectUrl: '{settings.CLERK_LOGOUT_REDIRECT_ROUTE}' }});
-            }});
-        """
+    if request.htmx:
```

Comment:
Because the 2 scrips are the same other than the listener, could reduce some duplication doing this:

```python
    event_name = 'htmx:load' if request.htmx else 'DOMContentLoaded'
    logout_script = air.Script(
        f"""
            document.addEventListener('{event_name}', async () => {{
                if (!window.Clerk) return;

                await window.Clerk.load();

                // Sign out on the client side (clears cookies/tokens)
                await window.Clerk.signOut({{ redirectUrl: '{settings.CLERK_LOGOUT_REDIRECT_ROUTE}' }});
            }});
        """
    )
```